src/main/resources/logback-spring.xml

```diff
@@ -48,7 +48,7 @@
         <appender-ref ref="consoleAppender"/>
     </logger>
 
-    <root level="DEBUG">
+    <root level="INFO">
```

Comment:
is this deliberate - we normally run on `DEBUG` for all services

Reply:
All work why there was so much noise in DEBUG mode and correct